wire/proto.go

```diff
@@ -45,3 +45,14 @@ type RefLine struct {
 	NewRef  string `json:"newref"`
 	RefName string `json:"refname"`
 }
+
+// CommitListItem represents a subset of information from a git commit
+type CommitListItem struct {
```

Comment:
`CommitListItem` reads a bit funny, because that seems like a long way to say `Commit`, maybe that, although of course it is not really a git commit object, so maybe `CommitSummary`. Not sure that is any better ;)